assets/sass/components/global/_googlesitekit-gathering-data-notice.scss

```diff
@@ -19,54 +19,47 @@
 .googlesitekit-plugin {
 
 	.googlesitekit-gathering-data-notice {
-		position: relative;
-		text-align: center;
+		text-transform: lowercase;
 
-		p {
-			color: $c-jumbo;
-			font-size: 16px;
-			line-height: 19px;
-			margin: 0;
-			padding: 0 0 20px;
-			text-transform: lowercase;
+		&.googlesitekit-gathering-data-notice--has-style {
 
-			@media (min-width: $bp-tablet) {
-				font-size: 28px;
-				line-height: 58px;
-			}
-		}
+			div {
```

Comment:
@jimmymadon When changing the tag to be a `span`, let's add `display: block` to it.